manifests/ca.pp

```diff
@@ -117,22 +117,16 @@
     group   => $group_to_set,
   }
 
-  # directory shared with openvpn::server
-  ensure_resource(file, "/etc/openvpn/${name}", {
-    ensure  => directory,
-    mode    => '0750',
-  })
-
   exec { "copy easy-rsa to openvpn config folder ${name}":
     command => "/bin/cp -r ${openvpn::params::easyrsa_source} /etc/openvpn/${name}/easy-rsa",
     creates => "/etc/openvpn/${name}/easy-rsa",
-    notify  => Exec["fix_easyrsa_file_permissions_${name}"],
     require => File["/etc/openvpn/${name}"],
   }
 
-  exec { "fix_easyrsa_file_permissions_${name}":
-    refreshonly => true,
-    command     => "/bin/chmod 750 /etc/openvpn/${name}/easy-rsa/*",
+  file { "/etc/openvpn/${name}/easy-rsa":
```

Comment:
This is about changing the permissions of the scripts directly inside `.../easy-rsa/`, see the asterisk in the command. With this new implementation, it would not change the permissions of the script but of the folder only.

Maybe a better solution would be to change the permissions of the used scripts only: `vars`, `clean-all`, `,build-dh` and `pkitool`. What do you think?